### PR TITLE
Start the API endpoints documentation with versioning

### PR DESCRIPTION
**Pull request by brouberol**

### CODE REVIEW

**Comment by ametaireau** [comment]
Works for me. Thanks.

**Comment by ametaireau** [comment]
We could add another information (a warning) telling that all requests in the documentation don't include the prefix to ease readability.

**Comment by brouberol** [comment]
Yep, good idea.

File: docs/api/versioning.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Each non retro-compatible API change will imply the major version number to be incremented.
+Everything will be made to avoid retro incompatible changes.
+
+The ``/`` endpoint will redirect to the last API version.
+
+
+.. warning::
+
+    The version prefix will be **implied** throughout the rest of the API reference, to improve readability. For example, the ``/`` endpoint should be understood as ``/v0/``.
```
Comment (ametaireau) [inline docs/api/versioning.rst]: This should be wrapped to 80 chars :)

Comment (brouberol) [inline docs/api/versioning.rst]: Only the warnings/notes/etc, or every line in the doc?

**Comment by ametaireau** [comment]
Thanks!

**Pull request merged**